lua/telescope/builtin/files.lua

```diff
@@ -49,7 +49,13 @@ files.live_grep = function(opts)
 
       prompt = escape_chars(prompt)
 
-      return flatten { vimgrep_arguments, prompt, opts.search_dirs or '.' }
+      local default_search_dir = nil
+
+      if vim.fn.has('win32') == 1 then
```

Comment:
I kinda like `os_sep == '\\'` more and it should be faster :laughing: 

Otherwise LGTM